src/components/gallery/HdGalleryOverlay.vue

```diff
@@ -60,8 +63,18 @@ export default {
         this.$refs.gallery.goNext();
       } else if (['Left', 'ArrowLeft'].includes(e.key)) {
         this.$refs.gallery.goPrev();
+      } else if (e.key === 'Escape') {
```

Comment:
I believe we should also check for `Esc` (Used by IE and Edge)
```suggestion
      } else if (['Esc', 'Escape'].includes(e.key)) {
```

Reply:
Nice catch, thanks!